text/chapter5.md

```diff
@@ -456,6 +456,14 @@ In this case, `Watt` is the _type constructor_ and `MakeWatt` is the _data const
 
 Another application of newtypes is to attach different _behavior_ to an existing type without changing its representation at runtime. We cover that use case in the next chapter when we discuss _type classes_.
 
+## Exercises
+
+1. (Easy) Define a `calculateWattage` function using the above definitions `Amp`, `Volt`, and `Watt`:
+```haskell
+calculateWattage :: Amp -> Volt -> Watt
+```
+that calculates a wattage in Watts as the product of the given current in Amps and voltage in Volts.
```

Comment:
Don't know if this makes things any clearer. Up to your discretion. Will merge either way.
```suggestion
that calculates a wattage in `Watt`s as the product of the given current in `Amp`s and voltage in `Volt`s.
```